webapp/src/Service/RejudgingService.php

```diff
@@ -75,29 +48,25 @@ public function __construct(
      * @param string          $reason           Reason for this rejudging
      * @param array           $judgings         List of judgings to rejudging
      * @param bool            $autoApply        Whether the judgings should be automatically applied.
-     * @param int             $repeat
-     * @param Rejudging|null  $repeatedRejudging
      * @param array          &$skipped          Returns list of judgings not included.
      * @param callable|null   $progressReporter If set, report progress using this callback. Will get two values:
      *                                          - the progress as an integer
      *                                          - the log to display
      *
-     * @return Rejudging|null
      */
     public function createRejudging(
         string $reason,
         int $priority,
         array $judgings,
         bool $autoApply,
-        int $repeat,
+        ?int $repeat,
```

Comment:
This was not in the phpdoc.

Reply:
Nope but below we do an isset() on it so I assume it can be null.